iib/workers/tasks/utils.py

```diff
@@ -842,15 +800,20 @@ def _get_binary_versions():
     :return: String with all binary used and their version
     :rtype: str
     """
-    opm_version_cmd = ['opm', 'version']
+
     podman_version_cmd = ['podman', '-v']
     buildah_version_cmd = ['buildah', '-v']
 
-    opm_version = run_cmd(opm_version_cmd, exc_msg='Failed to get opm version.').strip()
     podman_version = run_cmd(podman_version_cmd, exc_msg='Failed to get podman version.').strip()
     buildah_version = run_cmd(buildah_version_cmd, exc_msg='Failed to get buildah version.').strip()
-
-    return f"opm {opm_version}\n{podman_version}\n{buildah_version}"
+    binary_versions = f"{podman_version}\n{buildah_version}"
+    opms_present = [f for f in os.listdir('/usr/bin') if re.match(r'opm', f)]
```

Comment:
I think we can leverage `iib_ocp_opm_mapping`. 
Something like: 
```python
opms_present = set(iib_ocp_opm_mapping.values()) if iib_ocp_opm_mapping is not None else ['opm']
```

Reply:
updated